plugin/src/main/java/io/deephaven/plugin/type/EchoObjectType.java

```diff
@@ -0,0 +1,38 @@
+package io.deephaven.plugin.type;
+
+import com.google.auto.service.AutoService;
+
+import java.nio.ByteBuffer;
+
+/**
+ * An {@link ObjectType object type plugin} named {@value NAME} that echos back the received contents back to the
+ * client. When {@link #compatibleClientConnection(Object, MessageStream)} is called, an empty payload is sent to the
+ * client.
+ *
+ * <p>
+ * The only object that is compatible with this plugin is {@link #INSTANCE}.
+ */
+@AutoService(ObjectType.class)
+public final class EchoObjectType extends ObjectTypeBase {
```

Comment:
Do we intend this to be checked in? It probably should be in its own test jar, but not automatically part of every distribution?

Reply:
I was thinking it could be generally useful for production deployments that wanted to test new plugin development; but if we don't want to promote it as such, happy to refactor.